addons/crm_project_issue/__openerp__.py

```diff
@@ -12,8 +13,8 @@
 Link module to map leads to issues
         """,
     'data': [
-        'project_issue_view.xml'
+        'wizard/crm_lead2projectissue_wizard_view.xml',
+        'views/crm_lead_views.xml'
```

Comment:
@WHPODOO This time you are wrong. Just go through crm_lead_views.xml file you can get convert_lead2projectissue_wizard_action called there and that record is in wizard file. So the wizard file should be placed first here. 
![636x460design_01](https://cloud.githubusercontent.com/assets/12712586/12085553/4ec1a35c-b2e7-11e5-8ee9-f6871bbaa04e.jpg)

Think twice before commenting on any PR.
And one more thing I have learned A,B,C,D in my Nursery.  :D